docs/docusaurus/docs/cloud/users/manage_users.md

```diff
@@ -38,19 +38,15 @@ With Admin permissions, you can add users, edit organization roles, and delete u
 
 ## Create a user access token
```

Comment:
Do we describe the use cases for user vs org access tokens anywhere?

Reply:
@annabarr as the documentation doesn't currently specify that users generate organization tokens, the use cases for the different token types are not provided. The UI includes this text for the organization token:

_This token provides GX Cloud Editor permissions to shared systems, such as a scheduled pipeline run._

and this text for the User token:

_This token provides GX Cloud Editor permissions to users._

This is the text in the documentation:

_You'll need your access token and organization ID to set your access credentials._

If you think providing the use cases for the different token types would be helpful, let me know, and I'll create the text.